CHANGELOG.md

```diff
@@ -5,8 +5,11 @@
 * None.
 
 ##### Enhancements
-
-* None.
+--------------------------------------------------------------------------------
```

Comment:
Don't need this line, see style in the rest of the file.  Could drop the 'to jazzy' part too, I think the context is clear.

Reply:
✅